src/replication.c

```diff
@@ -3006,6 +3019,13 @@ void syncWithMaster(connection *conn) {
             if (err) goto write_error;
         }
 
+        replconf_rdb_no_compress = 0;
```

Comment:
```suggestion
        /* If we are not going to save the RDB to disk, request that RDB
         * compression be disabled, which speeds up RDB delivery. */
        replconf_rdb_no_compress = 0;
```
maybe we can add a short comment to describe why we are doing this.